config/test.yaml

```diff
@@ -0,0 +1,5 @@
+postgres:
+  user: postgres
+  password: postgres
+  dbname: iterable_test
```

Comment:
Ugh. I'm definitely gonna have to make a bunch of changes to db/migrate after this PR gets in, but yeah. db/migrate_test sounds good.
Also, what's the point of having a test db?